services/agents/rds_exporter.go

```diff
@@ -0,0 +1,71 @@
+// pmm-managed
+// Copyright (C) 2017 Percona LLC
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Affero General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU Affero General Public License for more details.
+//
+// You should have received a copy of the GNU Affero General Public License
+// along with this program. If not, see <https://www.gnu.org/licenses/>.
+
+package agents
+
+import (
+	"fmt"
+	"sort"
+
+	"github.com/AlekSi/pointer"
+	"github.com/percona/pmm/api/agentpb"
+	"github.com/percona/pmm/api/inventorypb"
+	"gopkg.in/yaml.v2"
+
+	"github.com/percona/pmm-managed/models"
+)
+
+// rdsExporterConfig returns desired configuration of rds_exporter process.
+func rdsExporterConfig(node *models.Node, exporter *models.Agent) *agentpb.SetStateRequest_AgentProcess {
```

Comment:
That function should accepts several nodes and agents – all RDS instances that should be monitored from that host. We will start a single rds_exporter for one host/pmm-agent. That will allow rds_exporter to make a single API call to get data for several RDS instances.